src/tools/cephfs/cephfs-shell

```diff
@@ -1227,6 +1235,45 @@ sub-directories, files')
                 self.perror('max_files is not set', end='\n', apply_style=True)
                 pass
 
+    snap_parser = argparse.ArgumentParser(description='Snapshot Management')
+    snap_parser.add_argument('op', type=str, help='Snapshot operation.',
+                             choices=['create', 'delete'], nargs='?',
+                             default=None)
+    snap_parser.add_argument('name', type=str, help='Name of snapshot')
+    snap_parser.add_argument('dir', type=str, action=path_to_bytes,
+                             help='Directory for which snapshot '
+                                  'needs to be created or deleted')
+
+    @with_argparser(snap_parser)
+    def do_snap(self, args):
+        """
+        Snapshot management for the volume
+        """
+        snapdir = '.snap'
+        conf_snapdir = cephfs.conf_get('client_snapdir')
+        if conf_snapdir is not None:
+            snapdir = conf_snapdir
+        if args.op == 'create':
+            try:
+                if is_dir_exists(args.dir):
+                    cephfs.mkdir(os.path.join(args.dir, snapdir, args.name), 0o755)
+                else:
+                    self.poutput("error: no such directory '%s'" % args.dir)
```

Comment:
For printing errors, please use perror() instead of poutput().

Reply:
> For printing errors, please use perror() instead of poutput().

done.
@varshar16 @batrick please review the changes